src/panels/features/office/Club.js

```diff
@@ -57,31 +292,32 @@ export default class ClubCard extends Component {
         {this.props.isDesktop && <SplitCol width={125} maxWidth={125}></SplitCol>}
         <SplitCol>
           {this.props.isDesktop && <div style={{ marginBottom: "50px" }}></div>}
```

Comment:
Можно использовать одинарный тег.
```suggestion
          {this.props.isDesktop && <div style={{ marginBottom: "50px" }}/>}
```